babar-report/src/IO.vue

```diff
@@ -9,6 +9,10 @@
     <template v-if="isProcFSProfilerUsed">
     
       <b-container fluid>
+        <PlotTimeSeries title="Network bytes received/transmitted / sec" yAxis="bytes/sec" :series="series.netIO" />
+        <div class="explanation">
+          This graph shows the total number of bytes of data transmitted or received by all non-virtual interfaces by the entire process-tree as reported by the <kbd>/prod/net/dev</kbd> file.<br>
```

Comment:
There's a typo, it's `/proc`. I have made that typo everywhere else, I'll push a PR to fix the other ones ;)